packages/storefront-lwc/src/modules/commerce/lineitemtotal/lineitemtotal.html

```diff
@@ -0,0 +1,21 @@
+<!--
+    Copyright (c) 2020, salesforce.com, inc.
+    All rights reserved.
+    SPDX-License-Identifier: BSD-3-Clause
+    For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+-->
+
+<template>
+    <div class = "lineitemtotal">
```

Comment:
if we do not have any styling for lineitemtotal, then i dont think we need to have a class for that. Dont know if the team feels differently about it.

Reply:
similarly, i noticed a lot of un-needed classes that came over from legacy code because of copy-paste.

Reply:
good point. I removed a bunch of un-needed class names.